infrastructure/datastore/db_test.go

```diff
@@ -0,0 +1,180 @@
+package datastore
+
+import (
+	"reflect"
+	"taskmanager/domain/model"
+	"testing"
+)
+
+func Test_dbType_FindByID(t *testing.T) {
```

Comment:
Your test names could all become PascalCase. You don't need the underscores.

Reply:
I will rename those, actually those names were generated by VS Code test generation feature, it seems weird tbh